composer.json

```diff
@@ -27,7 +27,7 @@
   ],
   "autoload": {
     "psr-4": {
-      "Leean\\": "src/"
+      "Leanp\\": "src/"
```

Comment:
Yes.
All the Leanp instances are replaced by the plugin name when you run the generator.